src/parser/jobs/drk/modules/ResourceSimulator.js

```diff
@@ -121,6 +124,7 @@ const COMBO_GRIT_GENERATORS = {
 export default class Resources extends Module {
 	static handle = 'resourcesim'
 	static title = 'Resource Analyzer'
+	static collapsible = false
```

Comment:
This doesn't do anything - it was quickly replaced with `static displayMode` (`Module.ts:14`). If it looks good as is, though, probably don't need to change the mode.

Reply:
Ah, I was wondering why it didn't work, I figured I had something screwed up with my local branch along the same lines as why the GCD display wasn't right the other day.  

I'd rather mark this module as non-collapsible, since otherwise it collapses right at the end of the graph and doesn't make it very obvious that there's anything more available.  I might cut the spenders/generators panels in a future rework of the module, but as long as they're there, I think they need to display by default.

Reply:
Fair